scripts/amplfi_init.py

```diff
@@ -0,0 +1,168 @@
+from jsonargparse import ArgumentParser
+
+root = Path(__file__).resolve().parent.parent
+data_config = (root / "amplfi" / "law" / "datagen.cfg",)
+TUNE_CONFIGS = [
+    root / "projects" / "train" / "train" / "tune" / "tune.yaml",
+    root / "projects" / "train" / "train" / "tune" / "search_space.py",
+]
+
+
+TRAIN_CONFIGS = {
+    "similarity": [
+        root / "projects" / "train" / "configs" / "similarity" / "cbc.yaml"
+    ],
+    "flow": [root / "projects" / "train" / "configs" / "flow" / "cbc.yaml"],
+}
+
+
+def copy_configs(
+    path: Path,
+    configs: list[Path],
+):
+    """
+    Copy the configuration files to the specified directory for editing.
+
+    Any path specific configurations will be updated to point to the
+    correct paths in the new directory.
+
+    Args:
+        path:
+            The directory to copy the configuration files to.
+        configs:
+            The list of configuration files to copy.
+    """
+
+    path.mkdir(parents=True, exist_ok=True)
+    for config in configs:
+        dest = path / config.name
+        if config.name == "tune.yaml":
+            with open(config, "r") as f:
+                dict = yaml.safe_load(f)
+                dict["train_config"] = str(path / "cbc.yaml")
+
+            with open(dest, "w") as f:
+                yaml.dump(dict, f)
+        else:
+            shutil.copy(config, dest)
+
+
+def write_content(content: str, path: Path):
+    content = dedent(content).strip("\n")
+    with open(path, "w") as f:
+        f.write(content)
+
+    # make the file executable
+    path.chmod(0o755)
+    return content
+
+
+def create_runfile(
+    path: Path,
+    mode: Literal["flow", "similarity"],
+    pipeline: Literal["tune", "train"],
+    s3_bucket: Optional[Path] = None,
+):
+    # if s3 bucket is provided
+    # store training data and training info there
+    base = path if s3_bucket is None else s3_bucket
+
+    config = path / "datagen.cfg"
+    # make the below one string
+    data_cmd = f"LAW_CONFIG_FILE={config} poetry run "
+    data_cmd += f"--directory {root / 'amplfi' / 'law'} "
+    data_cmd += "law run amplfi.law.DataGeneration --workers 5\n"
+
+    train_root = root / "projects" / "train"
+    train_cmd = f"poetry run --directory {train_root} python "
+
+    if pipeline == "tune":
+        train_cmd += (
+            f"{train_root / 'train' / 'tune' / 'tune.py'} --config tune.yaml"
+        )
+    else:
+        cli = "similarity" if mode == "similarity" else "flow"
+        train_cmd += (
+            f"{train_root / 'train'  / 'cli' / f'{cli}.py'} "
+            "fit --config cbc.yaml"
+        )
+
+    content = f"""
+    #!/bin/bash
+    # Export environment variables
+    export AMPLFI_DATADIR={base}
+    export AMPLFI_OUTDIR={base}/training/
+    export AMPLFI_CONDORDIR={path}/condor
+
+    # launch the data generation pipeline;
+    # note that if you've made local code changes not in the containers
+    # you'll need to add the --dev flag!
```

Comment:
How is this used? `$ amplfi_init --dev`? Does the parser have this argument?

Reply:
This refers the `--dev` flag for the `law` tasks. Not as relevant here where we only use law for querying data.

Reply:
ok let's remove the lines then to avoid confusion